test/regex.test.ts

```diff
@@ -104,40 +104,40 @@ describe('Regex tests', () => {
       .check()
     ).to.have.property('satisfiable', true)
   })
-  
+
   it('should perform shrinking while maintaining regex pattern', () => {
     const pattern = /\d{3}-\d{2}-\d{4}/  // SSN format
     const original = '123-45-6789'
     const shrunk = fc.shrinkRegexString(original, pattern)
-    
+
     // Check that all shrunk values still match the pattern
     expect(shrunk.every(s => pattern.test(s))).to.be.true
-    
+
     // Check that some shrinking actually happened
     expect(shrunk.length).to.be.greaterThan(0)
-    
+
     // Check that shrunk values are in some way "simpler"
     // For numeric sequences, this often means "smaller numbers"
     const simplifiedExists = shrunk.some(s => {
       // Check if the shrunk value uses simpler digits (more 0's or 1's)
-      const countSimpleDigits = (str: string) => 
-        (str.match(/[01]/g) || []).length
-      
+      const countSimpleDigits = (str: string) =>
+        (str.match(/[01]/g) ?? []).length
+
       return countSimpleDigits(s) > countSimpleDigits(original)
     })
-    
+
     expect(simplifiedExists).to.be.true
   })
-  
+
   // Simplified validation test
   it('should validate proper email format', () => {
     expect(fc.scenario()
       .forall('email', fc.patterns.email())
       .then(({email}) => {
         // An email must contain exactly one @ symbol
-        return email.includes('@') && email.includes('.')
+        return (email as string).includes('@') && (email as string).includes('.')
```

Comment:
Not sure why email needs to be converted to string... I assumed an email already _acted_ as a string, but maybe this should be checked.